### DISCUSSION TITLE
Standard name for raw pb message/data in wrapper classes

### DISCUSSION THREAD

**Discussion by llucax** [frequenz-floss/frequenz-api-common#215] [Ideas]
Coming from https://github.com/frequenz-floss/frequenz-client-reporting-python/pull/16#discussion_r1548115921:

The PR calls it `_data_pb`, the SDK `_raw`. It would be nice to agree on some standard so all clients use the same, although it is not exposed so it is not that important.

Maybe `_pb_message` is the most accurate approach, or just `_pb` if we want to be extra terse, as it is only a private attribute.

**Comment by Marenz** [comment]
I vote for `_pb`
